api/recipient/recipientModel.js

```diff
@@ -36,34 +36,14 @@ const create = async (recipients) => {
       newRecipientsId = createdRecipients[0].recipient_id;
     });
 
-    return await findById(newRecipientsId);
+    return findById(newRecipientsId);
```

Comment:
Could have just called findAll here instead of calling findById here, and then findAll in that.

But I think I get what you're trying to do, it's easier for people to come into the project is they see a findBy vs trying to understand how findAll works.